packages/demo/usecase-demo/src/modules/DisclaimerModal.tsx

```diff
@@ -34,32 +33,36 @@ export function DevModal() {
   useEffect(() => {
     setStorageDevMode(isDevMode);
   }, [isDevMode]);
-  
+
   useEffect(() => {
```

Comment:
We can probably avoir this useEffect, and setting `StorageDisclaimerViewed` to true in the `onOpenChange` modal callback.

Reply:
done